resteasy-jaxrs/src/main/java/org/jboss/resteasy/core/MediaTypeMap.java

```diff
@@ -272,7 +272,7 @@ private CachedMediaTypeAndClass(Class clazz, MediaType mediaType)
       {
          this.clazz = new WeakReference(clazz);
          this.mediaType = mediaType;
-         int result = getClazz().hashCode();
+         int result = clazz.hashCode();

```

Comment:
Ah, sorry, I missed that the parameter for the constructor has the same name. So, at this point, I'm fine. Thanks.